cedar-policy-core/src/est/scope_constraints.rs

```diff
@@ -461,6 +467,8 @@ impl std::fmt::Display for ActionConstraint {
                 std::fmt::Display::fmt(aic, f)?;
                 Ok(())
             }
+            #[cfg(feature = "tolerant-ast")]
+            Self::ErrorConstraint => write!(f, "action"),
```

Comment:
This is not the same you were doing for other `Display` implementations.

Reply:
Good catch, will fix